README.md

```diff
@@ -108,6 +110,28 @@ You can restrict what fields to send by adding `api: false` to a specific schema
 }
 ```
 
+Or you can require only specific fields in the GET request by adding the query filters `includeFields` and `excludeFields`.
+
+Examples:
+`/api/v1/products?includeFields=type,slug,name`
+`/api/v1/products?excludeFields=type,slug,name`
+
+It is useless to use both `includeFields` and `excludeFields` in the same query, as `includeFields` has the priority over `excludeFields`.
+
+Excluded fields with `api: false` or `api: 'editPermissionRequired'` if appropriate will not be displayed of course, even if added to `includeFields`.
+Example of a response to `/api/v1/products?includeFields=type,slug,excludedFieldInSchema`
+
+```
+[
+  {
+    _id: 'whatever_id',
+    type: 'product',
+    slug: 'product-key-product',
+    _originalWidgets: {}
```

Comment:
`_originalWidgets` being here is really a bit of a bug, so please leave it out of the example and I'll work on getting it out of the response (: